### PR TITLE
Known Issues Page

### PR DESCRIPTION
**Pull request by nabobalis**
I wasn't really sure what angle to take with it.

Anyone have any ideas?

### CODE REVIEW

File: docs/known_issues.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+************
+Known Issues
+************
+
+This page documents common known issues with sunpy.
+"Issues" here is slighty more broad than just bugs but also oddities or specifics of how sunpy works.
+
+1. Do we have any?
```
Comment (nabobalis) [inline docs/known_issues.rst:8]:
```suggestion
```

**Comment by nabobalis** [comment]
Move to api subpart.

**Comment by nabobalis** [comment]
Add: https://github.com/sunpy/sunpy/issues/5447

File: docs/code_ref/index.rst [hunk 2]
```diff
@@ -7,18 +7,19 @@ Code Reference
 .. toctree::
    :maxdepth: 2
 
-   sunpy
    coordinates/index
    data
    database
    image
    io
+   known_issues
```
Comment (dstansby) [inline docs/code_ref/index.rst:15]: I feel like known issues and stability should go by themselves after the list of sub-modules?

Comment (nabobalis) [reply docs/code_ref/index.rst:15]: Good point, I sorted it since that's how I roll.

**Comment by nabobalis** [comment]
I've added two sections that will need reviewing.

File: docs/code_ref/known_issues.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Differences between ``anytim2tai`` and `astropy.time.Time`
+==============================================================
+
+* For dates on and after ``1972-01-01 00:00:00 UTC``, this format is equivalent to that returned by the ``anytim2tai`` routine in SSW.
+* For dates between ``1958-01-01 00:00:00 UTC`` and ``1972-01-01 00:00:00 UTC``, ``anytim2tai`` returns a constant value of 9 s difference between UTC and TAI time while `astropy.time.Time` returns a value of 0 s difference between UTC and TAI on ``1958-01-01 00:00:00``, with this difference increasing approximately linearly to 10 s on 1972-01-01 00:00:00, where the two approaches agree.
+* UT and TAI were synchronized by definition on ``1958 Jan 1 00:00:00``.
+  ``anytim2tai`` should return 0 at that time.
+* Even if ``anytim2tai`` is intentionally choosing to ignore the difference in lengths of UT seconds and TAI seconds prior to 1972, it should be returning 10 for pre-1972 times, not 9.
+  Its implementation acts as if there was a leap second added on 1972 Jan 1, but there was not.
```
Comment (ayshih) [inline docs/code_ref/known_issues.rst:15]:
```suggestion
Disagreement between `astropy.time.Time` and SSW for pre-1972 dates
===================================================================

Conversions between time scales for pre-1972 dates may not agree between `astropy.time.Time` and SSW.
Prior to 1972, the length of a Universal Time (UT) second was different from the length of an SI second, and `astropy.time.Time` correctly handles this difference.
However, SSW does not (at least in the commonly used routines), and thus SSW conversions for pre-1972 dates between UT and International Atomic Time (TAI, based on SI seconds) can be incorrect by multiple seconds.
The SSW routine ``utc2tai`` notes that its output is invalid for pre-1972 dates, but other SSW routines are missing such a disclaimer.
```

**Review by Cadair — APPROVED** [review]

**Pull request merged**